packages/api/controllers/hackathon_register_controller.go

```diff
@@ -51,9 +51,18 @@ func RegisterTeamMember(c *fiber.Ctx) error {
 		ShareInfoWithSponsors:          member.ShareInfoWithSponsors,
 		WantJobOffers:                  member.WantJobOffers}
 
-	// numberOfTotalParticipants := GetTotalNumberOfParticipants()
 	numberOfTeams, _ := GetNumberOfHackathonTeams(ctx)
 
+	isEligeble, getNubmerErr := IsTotalNumberOfParticipantsEligeble(c)
```

Comment:
I did it in this way, because when I called the previous function which as you can see returned status codes, but when called here I didn't get them, so I decided to make it with bool and arr. In that way it is going to return the proper status code both here and when CheckRegistrationAvailability is called on GET to `/api/hackathon/register/available`

Reply:
Second of all, you should be able to pass down the request context as follows:

`/api/hackathon/register` -> `/api/hackathon/register/availability` -> check returned interface

I bet there is a way to check the return value